README.md

```diff
@@ -61,8 +61,22 @@ cd bedrock-claude-chat
 chmod +x bin.sh
 ./bin.sh
 ```
+### Optional Parameters
 
-- After about 30 minutes, you will get the following output, which you can access from your browser
+You can now specify the following parameters during deployment to enhance security and customization:
+
+- **--disable-self-register**: Disable self-registration (default: enabled). If this flag is set, you will need to create all users on cognito and it will not allow users to self register their accounts.
+- **--ipv4-ranges**: Comma-separated list of allowed IPv4 ranges. (default: allow all ipv4 addresses)
+- **--ipv6-ranges**: Comma-separated list of allowed IPv6 ranges. (default: allow all ipv6 addresses)
+- **--allowed-signup-email-domains**: Comma-separated list of allowed email domains for sign-up. (default: no domain restriction)
+
+#### Example command with parameters:
+
+```sh
+./bin.sh --disable-self-register --ipv4-ranges "192.0.2.0/25,192.0.2.128/25" --ipv6-ranges "2001:db8:1:2::/64,2001:db8:1:3::/64" --allowed-signup-email-domains "example.com,anotherexample.com"
```

Comment:
Are the addresses anonimized? (looks like a real address?)

Reply:
Those are addresses defined to be used in public documentations.

Follow RFCs
https://datatracker.ietf.org/doc/html/rfc3849
https://datatracker.ietf.org/doc/html/rfc5737